packages/client/hmi-client/src/components/model/model-parts/tera-model-part-filter.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<aside class="flex gap-2">
+		<Button
+			v-if="!isEmpty(getErrors())"
+			size="small"
+			:disabled="isFilterWarning"
+			:severity="!isFilterError ? 'danger' : 'secondary'"
+			@click.stop="toggleFilterSeverity(ModelErrorSeverity.ERROR)"
+		>
+			{{ !isFilterError ? 'Filter' : 'Unfilter' }} {{ getErrors().length }} errors
```

Comment:
```suggestion
			{{ isFilterError ? 'Unfilter': 'Filter'  }} {{ getErrors().length }} errors
```

Less negation is usually clearer.  Similar for other ternary operators like this